<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>book_form</class>
 <widget class="QDialog" name="book_form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>530</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>600</width>
    <height>530</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>600</width>
    <height>530</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>541</width>
     <height>451</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="0">
     <widget class="QPushButton" name="button_remvBook">
      <property name="text">
       <string>Remove Book</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QComboBox" name="cbox_findBook"/>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="edit_findBook"/>
    </item>
    <item row="3" column="0" colspan="2">
     <widget class="QTableView" name="tableBooks"/>
    </item>
    <item row="1" column="1">
     <widget class="QPushButton" name="button_addBook">
      <property name="text">
       <string>Add Book</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
